include/circt/Dialect/Moore/MooreOps.td

```diff
@@ -1006,12 +1008,21 @@ def StructExtractOp : MooreOp<"struct_extract"> {
     ```
     See IEEE 1800-2017 § 7.2.1 "Assigning to structures".
   }];
-  let arguments = (ins StrAttr:$memberName, UnpackedType:$input);
+  let arguments = (ins StrAttr:$fieldName, UnpackedType:$input);
```

Comment:
Can you also make the `$input` an `AnyStructType`?

Reply:
Since we already have a verifier that verify the type of inputs, we do not need to have this exact type declartion. Types like anyStructType could be replaced to unpacked type.